cmd/mod.go

```diff
@@ -124,7 +126,7 @@ new internal ids is printed.`,
 		}()
 
 		// get the job(s) user wishes to modify
-		jobs := getJobs(jq, jobqueue.JobStateDeletable, cmdAll, 0, false, false)
+		jobs := getJobs(ctx, jq, jobqueue.JobStateDeletable, cmdAll, 0, false, false)
```

Comment:
Probably a few of these private methods in cmd package no longer need ctx; go through and check them all.